### DISCUSSION TITLE
Can't build the sample code

### DISCUSSION THREAD

**Discussion by daviddrell** [microsoft/WinUI-Gallery#1425] [General]
I get a number of errors when building in VS 2022.

I have updated all the nuget packages.

Below please find all the VS2022 version info followed by the list of build errors.

###############  VS 2022 Info ###################################

Windows Software Development Kit: 10.0.19041.685

Microsoft Visual Studio Community 2022
Version 17.6.5
VisualStudio.17.Release/17.6.5+33829.357
Microsoft .NET Framework
Version 4.8.09037

Installed Version: Community

Visual C++ 2022   00482-90000-00000-AA907
Microsoft Visual C++ 2022

ASP.NET and Web Tools   17.6.326.62524
ASP.NET and Web Tools

Azure App Service Tools v3.0.0   17.6.326.62524
Azure App Service Tools v3.0.0

Azure Functions and Web Jobs Tools   17.6.326.62524
Azure Functions and Web Jobs Tools

C# Tools   4.6.0-3.23259.8+c3cc1d0ceeab1a65da0217e403851a1e8a30086a
C# components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Common Azure Tools   1.10
Provides common services for use by Azure Mobile Services and Microsoft Azure Tools.

Cookiecutter   17.0.23087.1
Provides tools for finding, instantiating and customizing templates in cookiecutter format.

GitExtensions   1.0
Git Extensions is a graphical user interface for Git that allows you to control Git without using the command-line

GitHub Copilot   1.100.0.0 (v1.100.0.0@6ff082509)
GitHub Copilot is an AI pair programmer that helps you write code faster and with less work.

GitHub Copilot Agent   1.100.306 (v1.100.0)

Linux Core Dump Debugging   1.0.9.33801
Enables debugging of Linux core dumps.

Microsoft JVM Debugger   1.0
Provides support for connecting the Visual Studio debugger to JDWP compatible Java Virtual Machines

NuGet Package Manager   6.6.0
NuGet Package Manager in Visual Studio. For more information about NuGet, visit https://docs.nuget.org/

NVIDIA CUDA 11.7 Wizards   11.7
Wizards to create new NVIDIA CUDA projects and source files.

NVIDIA Nsight Visual Studio Edition   2022.2.0.22095
NVIDIA Nsight Visual Studio Edition provides tools for GPGPU and graphics development.  Copyright © NVIDIA 2010 - 2022.
•Direct3D® and DirectX® are registered trademarks of Microsoft Corporation in the United States and/or other countries.
•Microsoft Detours is used under the Professional license (http://research.microsoft.com/en-us/projects/detours/).
•Gardens Point Parser Generator Copyright 2005 Queensland University of Technology (QUT). All rights reserved.
•Icons from Axialis Software used under the licensing terms found here: www.axialis.com
•NLog Copyright © 2004-2006 Jaroslaw Kowalski (jaak@jkowalski.net)
•zlib and libpng used under the zlib/libpnc license (http://opensource.org/licenses/Zlib) 
•Breakpad Copyright ©2006, Google Inc. All rights reserved.
•The OpenGL Extension Wrangler Library
Copyright ©2008-2016, Nigel Stewart (nigels@users.sourceforge.net), Copyright ©2002-2008, Milan Ikits (milan.ikits@ieee.org), Copyright ©2002-2008, Marcelo E. Magallon (mmagallo@debian.org), Copyright ©2002, Lev Povalahev. 
All rights reserved. 
•LIBSSH2 Copyright ©2004-2007 Sara Golemon (sarag@libssh2.org), Copyright ©2005,2006 Mikhail Gusarov (dottedmag@dottedmag.net),Copyright ©2006-2007 The Written Word, Inc.,Copyright ©2007 Eli Fant (elifantu@mail.ru),Copyright ©2009-2014 Daniel Stenberg., Copyright ©2008, 2009 Simon Josefsson.
All rights reserved. 
•Protobuf Copyright ©2014, Google Inc. All rights reserved.
•xxHASH Library Copyright ©2012-2014, Yann Collet. All rights reserved. 
•FMT Copyright ©2012 - 2016, Victor Zverovich 
•Font Awesome Copyright 2018 Fonticons, Inc. 
•ELF Definitions Copyright (c) 2010 Joseph Koshy, All rights reserved.

Warning: This computer program is protected by copyright law and international treaties. Unauthorized reproduction or distribution of this program, or any portion of it, may result in severe civil and criminal penalties, and will be prosecuted to the maximum extent possible under the law.

NVIDIA Nsight Visual Studio Edition - CUDA support   2022.2.0.22095
NVIDIA Nsight Visual Studio Edition - CUDA support provides tools for CUDA development and debugging.

Python - Django support   17.0.23087.1
Provides templates and integration for the Django web framework.

Python - Profiling support   17.0.23087.1
Profiling support for Python projects.

Python with Pylance   17.0.23087.1
Provides IntelliSense, projects, templates, debugging, interactive windows, and other support for Python developers.

Razor (ASP.NET Core)   17.6.0.2327201+a6a61fdfa748eaa65aab53dab583276e26af4a3e
Provides languages services for ASP.NET Core Razor.

SQL Server Data Tools   17.6.13.0
Microsoft SQL Server Data Tools

Test Adapter for Boost.Test   1.0
Enables Visual Studio's testing tools with unit tests written for Boost.Test.  The use terms and Third Party Notices are available in the extension installation directory.

Test Adapter for Google Test   1.0
Enables Visual Studio's testing tools with unit tests written for Google Test.  The use terms and Third Party Notices are available in the extension installation directory.

TypeScript Tools   17.0.20329.2001
TypeScript Tools for Microsoft Visual Studio

Visual Basic Tools   4.6.0-3.23259.8+c3cc1d0ceeab1a65da0217e403851a1e8a30086a
Visual Basic components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Visual C++ for Cross Platform Mobile Development (Android)   17.0.33606.364
Visual C++ for Cross Platform Mobile Development (Android)

Visual C++ for Linux Development   1.0.9.33801
Visual C++ for Linux Development

Visual F# Tools   17.6.0-beta.23174.5+0207bea1afae48d9351ac26fb51afc8260de0a97
Microsoft Visual F# Tools

Visual Studio IntelliCode   2.2
AI-assisted development for Visual Studio.

###############  Build Errors ###################################

Severity	Code	Description	Project	File	Line	Suppression State
Error		Path 'D:\workspace\metavi\Windows-Machine-Learning\Samples\WinMLSamplesGallery\WinMLSamplesGalleryNative\bin\neutral\WinMLSamplesGalleryNative.winmd' is not a file or directory	WinMLSamplesGalleryNative.Interop	D:\workspace\metavi\Windows-Machine-Learning\Samples\WinMLSamplesGallery\WinMLSamplesGalleryNative.Interop\EXEC	1	
Error	MSB3073	The command "C:\Users\david\.nuget\packages\microsoft.windows.cswinrt\2.0.4\cswinrt.exe -verbose -in local -in "D:\workspace\metavi\Windows-Machine-Learning\Samples\WinMLSamplesGallery\WinMLSamplesGalleryNative\bin\neutral\WinMLSamplesGalleryNative.winmd" "D:\workspace\metavi\Windows-Machine-Learning\Samples\WinMLSamplesGallery\packages\Microsoft.AI.MachineLearning.1.11.0\winmds\Microsoft.AI.MachineLearning.winmd" -out "D:\workspace\metavi\Windows-Machine-Learning\Samples\WinMLSamplesGallery\WinMLSamplesGalleryNative.Interop\Generated Files" -include WinMLSamplesGalleryNative" exited with code 1.	WinMLSamplesGalleryNative.Interop	D:\workspace\metavi\Windows-Machine-Learning\Samples\WinMLSamplesGallery\WinMLSamplesGalleryNative.Interop\WinMLSamplesGalleryNative.Interop.csproj	32	
Error		This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is ..\packages\Microsoft.Windows.ImplementationLibrary.1.0.220201.1\build\native\Microsoft.Windows.ImplementationLibrary.targets.	WinMLSamplesGalleryNative	D:\workspace\metavi\Windows-Machine-Learning\Samples\WinMLSamplesGallery\WinMLSamplesGalleryNative\WinMLSamplesGalleryNative.vcxproj	267	
Error		Path 'D:\workspace\metavi\Windows-Machine-Learning\Samples\WinMLSamplesGallery\WinMLSamplesGalleryNative\bin\x64\Debug\WinMLSamplesGalleryNative.winmd' is not a file or directory	WinMLSamplesGalleryNative.Interop	D:\workspace\metavi\Windows-Machine-Learning\Samples\WinMLSamplesGallery\WinMLSamplesGalleryNative.Interop\EXEC	1	
Error	MSB3073	The command ""C:\Users\david\.nuget\packages\microsoft.windows.cswinrt\2.0.4\cswinrt.exe" @"D:\workspace\metavi\Windows-Machine-Learning\Samples\WinMLSamplesGallery\WinMLSamplesGalleryNative.Interop\obj\x64\Debug\net6.0-windows10.0.19041.0\Generated Files\CsWinRT\cswinrt.rsp"" exited with code 1.	WinMLSamplesGalleryNative.Interop	C:\Users\david\.nuget\packages\microsoft.windows.cswinrt\2.0.4\build\Microsoft.Windows.CsWinRT.targets	195	
Warning		Path 'D:\workspace\metavi\Windows-Machine-Learning\Samples\WinMLSamplesGallery\WinMLSamplesGalleryNative\bin\x64\Debug\WinMLSamplesGalleryNative.winmd' is not a file or directory	WinMLSamplesGallery	D:\workspace\metavi\Windows-Machine-Learning\Samples\WinMLSamplesGallery\WinMLSamplesGallery\EXEC	1	
Warning	MSB3073	The command ""C:\Users\david\.nuget\packages\microsoft.windows.cswinrt\2.0.4\cswinrt.exe" @"D:\workspace\metavi\Windows-Machine-Learning\Samples\WinMLSamplesGallery\WinMLSamplesGallery\obj\x64\Debug\net6.0-windows10.0.19041.0\Generated Files\CsWinRT\cswinrt.rsp"" exited with code 1.	WinMLSamplesGallery	C:\Users\david\.nuget\packages\microsoft.windows.cswinrt\2.0.4\build\Microsoft.Windows.CsWinRT.targets	195	
Warning	WMC1006	Cannot resolve Assembly or Windows Metadata file 'D:\workspace\metavi\Windows-Machine-Learning\Samples\WinMLSamplesGallery\WinMLSamplesGalleryNative.Interop\bin\x64\Debug\net6.0-windows10.0.19041.0\WinMLSamplesGalleryNative.Interop.dll'	WinMLSamplesGallery	D:\workspace\metavi\Windows-Machine-Learning\Samples\WinMLSamplesGallery\WinMLSamplesGallery\WinMLSamplesGallery.csproj	1	
Error	MSB3923	Failed to download file "https://github.com/onnx/models/blob/main/vision/classification/densenet-121/model/densenet-9.onnx?raw=true".  Response status code does not indicate success: 404 (Not Found).	WinMLSamplesGallery	D:\workspace\metavi\Windows-Machine-Learning\Samples\WinMLSamplesGallery\WinMLSamplesGallery\WinMLSamplesGallery.csproj	192	
Error		Specified EntryPointExe 'WinMLSamplesGallery.exe' was not found in the project outputs.	WinMLSamplesGallery (Package)	C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\DesktopBridge\Microsoft.DesktopBridge.targets	745

**Comment by marcelwgn** [comment]
The error message looks like this is not related to the WinUI Gallery but rather this repository: https://github.com/microsoft/Windows-Machine-Learning/tree/master/Samples/WinMLSamplesGallery

If you tried to built that app and not the WinUI Gallery, someone on that repository will probably be able to help you; This repository is for the WinUI Gallery :)
